src/index.js

```diff
@@ -1,106 +1,107 @@
   requestFactory,
-  signin,
   scrape,
   saveBills,
-  saveFiles,
+  htmlToPDF,
+  createCozyPDFDocument,
   log
-} = require("cozy-konnector-libs");
-const cheerio = require('cheerio');
-const pdf = require('pdfjs')
-const html2pdf = require('./html2pdf')
+} = require('cozy-konnector-libs')
+const moment = require('moment')
+moment.locale('fr')
+const URL = require('url').URL
+const jar = require('request').jar()
 const request = requestFactory({
-  // the debug mode shows all the details about http request and responses. Very usefull for
-  // debugging but very verbose. That is why it is commented out by default
-  // debug: true,
-  // activates [cheerio](https://cheerio.js.org/) parsing on each page
-  cheerio: false,
-  // If cheerio is activated do not forget to deactivate json parsing (which is activated by
-  // default in cozy-konnector-libs
-  json: false,
-  // this allows request-promise to keep cookies between requests
-  jar: true
-});
-
-const jar = request.jar()
-
-const baseUrl = "https://www.zalando.fr";
-
-module.exports = new BaseKonnector(start);
-
-// The start function is run by the BaseKonnector instance only when it got all the account
-// information (fields). When you run this connector yourself in "standalone" mode or "dev" mode,
-// the account information come from ./konnector-dev-config.json file
+  cheerio: true,
+  jar
+})
+
+const baseURL = new URL('https://www.zalando.fr')
+
+module.exports = new BaseKonnector(start)
+
 async function start(fields) {
-  log("info", "Authenticating ...");
-  await authenticate(fields.login, fields.password);
-  log("info", "Successfully logged in");
-  // The BaseKonnector instance expects a Promise as return of the function
-  // log('info', 'Fetching the list of documents')
-  // const $ = await request(`${baseUrl}/index.html`)
-  // // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
-  // log('info', 'Parsing list of documents')
-  // const documents = await parseDocuments($)
-  //
-  // // here we use the saveBills function even if what we fetch are not bills, but this is the most
-  // // common case in connectors
-  // log('info', 'Saving data to Cozy')
-  // await saveBills(documents, fields.folderPath, {
-  //   // this is a bank identifier which will be used to link bills to bank operations. These
-  //   // identifiers should be at least a word found in the title of a bank operation related to this
-  //   // bill. It is not case sensitive.
-  //   identifiers: ['books']
-  // })
+  log('info', 'Authenticating ...')
+  await authenticate(fields.login, fields.password)
+  log('info', 'Successfully logged in')
+  log('info', 'Fetching the list of documents')
+  const $ = await request(`${baseURL.origin}/moncompte/commandes/`)
+  log('info', 'Parsing list of documents')
+  const documents = await parseDocuments($)
+  log('info', 'Saving data to Cozy')
+  await saveBills(documents, fields.folderPath, {
+    identifiers: ['zalando']
```

Comment:
You have to add the contentType: 'application/pdf' option to allow the connector to works with a real cozy. But you will also need the next version of cozy-konnector-libs since there is a bug : https://github.com/konnectors/libs/pull/277